tests/__tests__/appTest.jsx

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import { shallow, mount, render } from 'enzyme';
+import App from '../../client/src/components/app.jsx';
+
+test('should have fetched data in state', async () => {
+  var component = await mount(<App />);
```

Comment:
mount() [returns](https://airbnb.io/enzyme/docs/api/mount.html#mountnode-options--reactwrapper) a ReactWrapper object, not a Promise, and it is synchronous, so async/await is not necessary. You need to wrap it inside a Promise to use await.